changelog/953.feature.rst

```diff
@@ -1 +1,2 @@
-Refactor out particle tracking integrators from the ParticleTracker class
+Refactor out the `~plasmapy.simulation.particle_integrators.boris_push` tracking
+integrator algorithm from `~plasmapy.simulation.particletracker.ParticleTracker`.
```

Comment:
Sorry, this previous version reflected the previous scope of the pr better! Just part of learning to write good change log entries - we really need to look back on them before merging stuff.